bftengine/CMakeLists.txt

```diff
@@ -104,6 +104,8 @@ target_include_directories(corebft PUBLIC include/communication)
 target_include_directories(corebft PUBLIC include/bcstatetransfer)
 target_include_directories(corebft PUBLIC include/simplestatetransfer)
 target_include_directories(corebft PUBLIC include/metadatastorage)
+target_include_directories(corebft PUBLIC utils/include)
```

Comment:
This is not the correct way to use the utils library in cmake. `target_link_libraries` will automatically include it. We already have that, so this line should not be needed.

Reply:
It made me troubles before, but now it's OK.